UpperSchool.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="lowerschool.html">Lower School</a>
+      </li>
+    </ul>
+        <a href="upperschool.html">Upper School</a>
```

Comment:
This list item should appear before the closing `</ul>` tag and needs to be inside a `<li>` tagset